### ISSUE TITLE
lora transmit example issue

### ISSUE THREAD

**Issue by 13koma** [dernasherbrezon/sx127x#4]
Doing everything according to instructions for platformio, after flashing the code i go to monitor and see my esp32 board reboots continuously with this error -
ESP_ERROR_CHECK failed: esp_err_t 0x10a (ESP_ERR_INVALID_VERSION) at 0x400d112d
file: "src/main.c" line 85
func: app_main
expression: sx127x_create(spi_device, &device)

**Comment by dernasherbrezon** [comment]
Hey,

Can you double check if you have sx127x chip and not sx126x? They are different and have different chip version. Hence the error.

**Comment by 13koma** [comment]
I have rfm95w module which has sx1276 chip

**Comment by 13koma** [comment]
wow i'm sorry, my breakoutboard for esp was soldered badly, that was the problem

**Comment by gustavrsh** [comment]
Sorry for commenting on this issue. I am migrating to your library from [https://github.com/Inteform/esp32-lora-library](esp32-lora-library) and I'm having the exactly same issue.

The linked library initializes correctly. I'll try to debug the issue later today. There are some differences between the libraries in regards to spi transfers.

**Comment by dernasherbrezon** [comment]
Yes, this is the first call via SPI bus, so if you incorrectly initialised it, it will fail.

**Comment by gustavrsh** [comment]
> Yes, this is the first call via SPI bus, so if you incorrectly initialised it, it will fail.

Thank you for your response.

I identified the problem. Reset pin was not initialized and set to high. TX works without error, I'll later get another board and test RX. Thank you again as your code handles the device better with the presence of interrupts.
